apps/synapse-portal-framework/src/components/index.ts

```diff
@@ -1,4 +1,4 @@
-import RouteControlWrapper from './RouteControlWrapper'
+import ExploreWrapper from './Explore/ExploreWrapper'
```

Comment:
Component & file were already renamed, this was just an outdated reference that caused build errors